tests/jest/unit/app/modules/kit-library/assets/js/hooks/use-kit-call-to-action-updated-url.test.js

```diff
@@ -0,0 +1,15 @@
+import useKitCallToActionUpdatedUrl from '../../../../../../../../../app/modules/kit-library/assets/js/hooks/use-kit-call-to-action-updated-url.js';
+
+test( 'useKitCallToActionUpdatedUrl should return the url with utm_term and utm_content', () => {
+	// Arrange
+	const url = 'https://elementor.com/pro/?utm_source=kit-library&utm_medium=wp-dash&utm_campaign=gopro';
+	const kitTitle = 'Digital Marketing Studio';
+	const kitId = '6242d5f21475e1001dc794ae';
```

Comment:
test with invalid values

Reply:
I think that here I will want to leave it this way. there may be some kitID's that contains special characters.

Reply:
what about `null`s? non-string? arrays? object?
And also about the title, not only the ID.